src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -11,11 +11,11 @@ import {
   WizardStep,
   WizardStepFunctionType,
 } from '@patternfly/react-core';
-import { Link, Prompt, Redirect, useHistory, useRouteMatch } from 'react-router-dom';
+import { Link, Redirect, useHistory, useRouteMatch } from 'react-router-dom';
 import { UseQueryResult } from 'react-query';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
 import { useFormField, useFormState } from '@konveyor/lib-ui';
-
+import { Prompt } from '@app/common/components/Prompt';
```

Comment:
Since this is not coming from react-router, do you think "Prompt" is a descriptive enough name? With no knowledge of the component I wouldn't guess it has anything to do with navigation.

Reply:
Probably not, I didn't think too much into the name so far - was mostly just working toward a PF style direct replacement. How about a generic `RouteGuard`?

Reply:
I like RouteGuard!